api/definition/swagger.yaml

```diff
@@ -766,6 +766,12 @@ components:
       description: Determines if areas with no data are returned as empty arrays, the default is false.
       schema:
         type: boolean
+    latest_only:
+      in: query
+      name: latest_only
+      description: Set to true to get data for the latest date period only, default is false. This overrides the years parameter if set to true.
+      schema:
+        type: boolean
```

Comment:
Can you re-generate the FE api client. Just to make sure we haven't broken anything with the types. Should be ok by the looks of things.